app/editor/src/features/content/morning-report/filter-sections/FilteredContentSection.tsx

```diff
@@ -0,0 +1,134 @@
+import { ToggleGroup } from 'components/toggle-group';
+import { ToolBarSection } from 'components/tool-bar';
+import { InputOption } from 'features/content/tool-bar/sections/filter';
+import { useLookupOptions } from 'hooks';
+import React from 'react';
+import { FaEye, FaFilter, FaIcons, FaNewspaper } from 'react-icons/fa';
+import { filterEnabled } from 'store/hooks/lookup/utils';
+import { Col, FieldSize, IOptionItem, OptionItem, Row, Select } from 'tno-core';
+
+import { defaultReportFilter } from '../constants/defaultReportFilter';
+import { IMorningReportFilter } from '../interfaces';
+
+export interface IFilteredContentSection {
+  onFilterChange: (filter: IMorningReportFilter) => void;
+  filter: IMorningReportFilter;
+  onSearch: (filter: IMorningReportFilter) => void;
+}
+
+/** The filter content section of the morning report */
+export const FilteredContentSection: React.FC<IFilteredContentSection> = ({
+  onFilterChange,
+  filter,
+  onSearch,
+}) => {
+  const [{ productOptions: pOptions, sourceOptions }] = useLookupOptions();
+  const [productOptions, setProductOptions] = React.useState<IOptionItem[]>([]);
+
+  /** TODO: Filter needs to apply the query string to the filter when reloading - same issue exists on content list view **/
```

Comment:
Is this still an issue?

Reply:
Yes for the multi-select. I have another ticket regarding Morning Report filter bugs and may add this fix to that ticket. Just an issue on refresh